config/source_blockchain.go

```diff
@@ -104,18 +104,25 @@ func (s *SourceBlockchain) Validate(destinationBlockchainIDs *set.Set[string]) e
 			return fmt.Errorf("invalid blockchainID in configuration. error: %w", err)
 		}
 		if !destinationBlockchainIDs.Contains(dest.BlockchainID) {
-			return fmt.Errorf("configured source subnet %s has a supported destination blockchain ID %s that is not configured as a destination blockchain",
+			return fmt.Errorf(
+				"configured source subnet %s has a supported destination blockchain ID %s that is not configured as a destination blockchain", //nolint:lll
```

Comment:
AFAIK, the only other way to preserve these long strings is through concatenation, where we'd put each section on its own line. Not the cleanest, so I think the `nolint` tag is appropriate in these instances.